src/benchmarks/micro/libraries/System.Collections/CopyTo.cs

```diff
@@ -21,6 +21,7 @@ public class CopyTo<T>
         private T[] _array;
         private List<T> _list;
         private ImmutableArray<T> _immutablearray;
+        private ImmutableList<T> _immutableList;
```

Comment:
All collection shave the CopyTo method (e. g. dictionaries, sets, etc) but since the original file only went with lists and arrays I also took that approach. Happy to add the rest of the collections if desired